app/Models/SnipeModel.php

```diff
@@ -155,9 +156,19 @@ public function setStatusIdAttribute($value)
         $this->attributes['status_id'] = $value;
     }
 
-    //
-    public function getDisplayNameAttribute()
+    // This gets a little twitchy since *most* things have a property in the table called "name" (but users don't)
+    // AND we want to be able to use the actual display_name value from the database if it's set (usually via SCIM)
+    protected function displayNameAttribute(): Attribute
```

Comment:
Should this be `displayName()` without `Attribute`?

The [docs](https://laravel.com/docs/11.x/eloquent-mutators#defining-an-accessor) don't have the trailing `Attribute`.

Reply:
It seems to want it this way? When I remove that, I get ` Undefined property: App\Models\User::$display_name in /Users/snipe/Herd/snipe-it/app/Models/SnipeModel.php:166`